test/testHidDevice.js

```diff
@@ -0,0 +1,83 @@
+/*
+ * == BSD2 LICENSE ==
+ * Copyright (c) 2014, Tidepool Project
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the associated License, which is identical to the BSD 2-Clause
+ * License as published by the Open Source Initiative at opensource.org.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the License for more details.
+ *
+ * You should have received a copy of the License along with this program; if
+ * not, you can obtain one from Tidepool Project at tidepool.org.
+ * == BSD2 LICENSE ==
+ */
+
+/*
+ * IOET
+ * Test for new hidDevice facilities
+ * */
+
+/* global beforeEach, describe, it */
+
+var expect = require('salinity').expect;
+
+var hidDevice = require('../lib/hidDevice.js');
+
+describe('hidDevice.js', function(){
```

Comment:
I started the pattern of "exists" testing because I was building multiple things that needed to conform to an interface. But it feels like this has now become a pattern that make it look like there are some tests but without actually doing anything useful.

This is as much for Tidepool as IOET, but it seems like it would be nice to at least make a stab at putting some useful test code in place -- actually test SOMETHING. For a HID device, the test won't be able to go all the way to the device, but some of those functions at least could have something more meaningful in them.